### DISCUSSION TITLE
Registering button presses in PCSX2 with Pressure Sensitivity

### DISCUSSION THREAD

**Discussion by HendyStark** [nefarius/DsHidMini#189] [Support]
So i followed this guide on how to set the pressure sensitive buttons on PCSX2 (https://vigem.org/projects/DsHidMini/FAQ/#how-do-i-set-the-controllers-pressure-buttons-in-pcsx2). When i registered the buttons with Mute digital pressure checked, it won't register unless i really press the buttons hard, the guide don't mention anything about the need to press the button hard to register, but i don't really think about it, so i continue registering all the buttons following the guide, when i tested it in MGS2: Substance, pressing the buttons with pressure sensitivity don't really do anything unless i press the buttons hard just like when registering the buttons in PCSX2. I thought it was my DS3 suddenly acting up, so i switch back to SXS mode to test it in RPCS3 and open controller tester, it works just normal.

I go back to SDF mode and to PCSX2, and test the game again, still the same, need to press hard, i try to clear the button mapping and register the buttons again but still the same, need to press hard. So i try looking around in Pad 1 settings tab and in the configure binding of the buttons with pressure sensitivity are look like this by default after registering them 
![image](https://user-images.githubusercontent.com/99085044/152690955-8510223b-0efb-4cd1-b8c7-6f874e246597.png)
As you can see, it's in Slider with deadzone value at 0.200, i thought maybe the deadzone value is what causing this, so i change all the deadzone value to 0 for all buttons with pressure sensitivity and now it works normally.
![image](https://user-images.githubusercontent.com/99085044/152691211-5782efe4-997b-4f6a-b60a-0a37a9fef663.png)

Am i doing something wrong here? or am i missing something again from the guide?
Thank you!

**Comment by nefarius** [comment]
> When i registered the buttons with Mute digital pressure checked, it won't register unless i really press the buttons hard, the guide don't mention anything about the need to press the button hard to register,

That's because in none of my tests with my own controllers this became an issue 🙂 

> As you can see, it's in Slider with deadzone value at 0.200, i thought maybe the deadzone value is what causing this, so i change all the deadzone value to 0 for all buttons with pressure sensitivity and now it works normally.

Well if it works normally now I guess those are simply the settings that your particular setup requires, no?

So in essence: it works? So what else can I tell you 😁

**Reply by HendyStark** [reply]
i thought maybe there's something i'm doing wrong, but yeah, it works haha, thanks!
